### PR TITLE
Adding Community seeders/dns introducers

### PR DESCRIPTION
**Pull request by hoffmang9**
### Purpose:

Currently wallet and node use only `dns-introducer.chia.net` when searching in the primary manner for nodes with open ports. This adds an initial batch of dns introducer/seeders run by various individuals and entities around the world. After fixes in 1.6.2, this list is randomly selected from to spread this traffic. If one seeder doesn't reply, wallet and node will pick a next one to try from the random order. If others wish to be included the can follow the [Chia Seeder User's Guide](https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Seeder-User-Guide) and make a pull request against initial-config.yaml. Note that the list is sorted by primary domain name alphabetical order (e.g h9 before hoffmang.) This removes one centralizing point from this software.

### Current Behavior:
All dns/seeder requests go to `dns-introducer.chia.net`.

### New Behavior:
Requests to dns/seeder will now pick randomly from a list of community and CNI run seeders which increases decentralization.

### Testing Notes:
Each community seeder was hand tested before inclusion with both `dig chia.hoffmang.com` and `dig -t AAAA chia.hoffmang.com`.

### CODE REVIEW

**Review by arvidn — DISMISSED** [review]

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: chia/util/initial-config.yaml [hunk 1]
```diff
@@ -411,6 +411,10 @@ full_node:
   # If you modify this, please change the hardcode as well from FullNode.set_server()
   dns_servers:
     - "dns-introducer.chia.net"
+    - "chia.ctrlaltdel.ch"
+    - "chia-seeder.h9.com"
+    - "chia.hoffmang.com"
+    - "seeder.xchpool.org"
```
[the remark this answers is not in the document]
Comment (hoffmang9) [reply chia/util/initial-config.yaml]: Done

File: chia/util/initial-config.yaml [hunk 2]
```diff
@@ -495,6 +499,10 @@ wallet:
 
   dns_servers:
     - "dns-introducer.chia.net"
+    - "chia.ctrlaltdel.ch"
+    - "chia-seeder.h9.com"
+    - "chia.hoffmang.com"
+    - "seeder.xchpool.org"
```
[the remark this answers is not in the document]
Comment (hoffmang9) [reply chia/util/initial-config.yaml]: And done

[the remark this answers is not in the document]
Comment (hoffmang9) [reply chia/util/initial-config.yaml]: I didn't realize I could just accept your recommends in the web UI. Next time.

**Review by cmmarslender — APPROVED** [review]

**Review by arvidn — APPROVED** [review]

**Review by wallentx — CHANGES_REQUESTED** [review]
https://github.com/Chia-Network/chia-blockchain/blob/348d578a4b6d73e7a22007ea5e340f8c6034e012/chia/cmds/configure.py#L100-L183
This doesn't survive swapping between `chia configure -t t` and `chia configure -t f`

**Comment by hoffmang9** [comment]
That looks like a bug report for `chia configure testnet` and somewhat independent from this PR.

**Review by wallentx — APPROVED** [review]
Survivability concern separated to a new issue https://github.com/Chia-Network/chia-blockchain/issues/14505

**Pull request merged**